packages/openapi-v3/src/controller-spec.ts

```diff
@@ -348,3 +351,34 @@ export function getControllerSpec(constructor: Function): ControllerSpec {
   }
   return spec;
 }
+
+/**
+ * Describe the provided Model as a reference to a definition shared by multiple
+ * endpoints. The definition is included in the returned schema.
+ *
+ * @example
+ *
+ * ```ts
+ * const schema = {
+ *   $ref: '#/components/schemas/Product',
+ *   definitions: {
+ *     Product: {
+ *       title: 'Product',
+ *       properties: {
+ *         // etc.
+ *       }
+ *     }
+ *   }
+ * }
+ * ```
+ *
+ * @param modelCtor The model constructor (e.g. `Product`)
+ * @param options Additional options
```

Comment:
Please use `@param modelCtor - The model ...` and `@param options - Additional options`

Reply:
Will do.

It's annoying that VSCode is generating tsdoc comment in a format that's not recognized by api-extractor :(

Reply:
@raymondfeng How can we automate this part of review process, so that `npm test` fails when there are any tsdoc comments violating api-extractor requirements?